dvc/main.py

```diff
@@ -16,6 +17,16 @@
 
 logger = logging.getLogger("dvc")
 
+footer = (
```

Comment:
Also, let's call it
```suggestion
FOOTER = (
```
so it conforms to the naming conventions :slightly_smiling_face: `logger` from above does not, but that is a common practice for that specific application.